src/lib/stitching/vortex/stitching.ts

```diff
@@ -252,6 +270,28 @@ export const vortexStitchingEnvironment = (localSchema: GraphQLSchema) => ({
         },
       },
     },
+    AnalyticsPartnerSalesStats: {
+      total: {
+        fragment: gql`
+          ... on AnalyticsPartnerSalesStats {
+            totalCents
+          }
+        `,
+        resolve: (parent, args, _context, _info) =>
+          amount(_ => parent.totalCents).resolve({}, args),
+      },
+    },
+    AnalyticsPartnerSalesTimeSeriesStats: {
+      total: {
+        fragment: gql`
+          ... on AnalyticsPartnerSalesStats {
```

Comment:
Should this not be the following?

```suggestion
          ... on AnalyticsPartnerSalesTimeSeriesStats {
```

Reply:
You are correct! Thanks for catching that! 👍